### PR TITLE
Migrate login flow to use gRPC 

### PR DESCRIPTION
**Pull request by gracegoo-stripe**
This change updates the login flow to use the gPRC protocol and integrates the flow into the UI. 

https://user-images.githubusercontent.com/75757829/140431176-73c58014-cf37-4c80-9ab4-db07318cc990.mov

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -240,9 +244,52 @@ export class Commands {
...
+
+    daemonClient.login(new LoginRequest(), async (error, response) => {
+      if (error) {
+        vscode.window.showErrorMessage(`Failed to login. ${error.details}`);
+      } else if (response) {
+        const pairingCode = response.getPairingCode();
+        await vscode.window
+          .showInformationMessage(
+            `Your pairing code is ${pairingCode}. \n This pairing code verifies your authentication with Stripe.`,
+            ...['Authenticate from Dashboard'],
```
Comment (etsai-stripe) [inline src/commands.ts:259]: should we have an option to `cancel`?

**Review by etsai-stripe — APPROVED** [review]
LGTM! just one non blocking comment

File: src/commands.ts (hunk 1 above)
Comment (gracegoo-stripe) [reply src/commands.ts:259]: Sounds good to me, adding!

**Review by etsai-stripe — APPROVED** [review]
thanks for making the change!

**Pull request merged**